custom_components/meural/__init__.py

```diff
@@ -22,8 +25,20 @@ async def async_setup(hass: HomeAssistant, config: dict):
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     """Set up Meural from a config entry."""
+    if "email" not in entry.data:
+        _LOGGER.warning("Authentication changed. Please set up Meural again")
+        return False
+
+    def token_update_callback(token):
+        _LOGGER.warning("Token changed. Updating config entry.")
```

Comment:
This should be `debug` logging at most.
```suggestion
        _LOGGER.debug("Token changed. Updating config entry.")
```